src/schema/datatypes/anat.yml

```diff
@@ -0,0 +1,44 @@
+group1:
+  datatypes:
+    - anat
```

Comment:
May be a folder name should be changed to not conflict with `datatypes` which are listed within files -- it should be reflective of the filenames in the folder which do not necessarily correspond to datatypes (which are listed within).

Reply:
Also "datatype" term is not defined nicely in BIDS. Related #152 ...
may be this field is actually the `modalities`? ;-)

Reply:
Did pybids switch from modality to datatype for this? I was just planning to go with what pybids has if it's not explicit in the specification.

Reply:
TL;DR: I was wrong - `modalities` would be wrong and `datatypes` indeed correct:
```
src/05-derivatives/02-common-data-types.md-```Text
src/05-derivatives/02-common-data-types.md-<pipeline_name>/
src/05-derivatives/02-common-data-types.md-    sub-<participant_label>/
src/05-derivatives/02-common-data-types.md:        <datatype>/
src/05-derivatives/02-common-data-types.md-            <source_keywords>[_space-<space>][_desc-<label>]_<suffix>.<ext>
```
I will add it now to #152

Reply:
doh -- just now mentioned it was from common-derivatives branch/PR, but I think it holds generally.

Reply:
I must stop for today for the sake of humanity and myself: "Data type" is defined within that glossary already ;-) Sorry for all the noise